src/views/splash/splash.jsx

```diff
@@ -344,21 +342,23 @@ var Splash = injectIntl(React.createClass({
         return (
             <div className="splash">
                 {this.shouldShowEmailConfirmation() ? [
-                    <DropdownBanner key="confirmedEmail"
-                            className="warning"
-                            onRequestDismiss={this.handleDismiss.bind(this, 'confirmed_email')}>
+                    <DropdownBanner
+                        key="confirmedEmail"
+                        className="warning"
+                        onRequestDismiss={this.handleDismiss.bind(this, 'confirmed_email')}
+                    >
```

Comment:
I don't get to comment on unchanged lines, but shouldn't the 'Confirm your email... to enable sharing', just below, be wrapped in a FormatHTMLMessage to be translated?